components/edit-collective/sections/UserSecurity.js

```diff
@@ -58,6 +58,15 @@ class UserSecurity extends React.Component {
     };
 
     this.setPassword = this.setPassword.bind(this);
+    this.componentMounted = false;
+  }
+
+  componentDidUpdate() {
+    if (window.location.hash && !this.componentMounted) {
+      this.componentMounted = true;
```

Comment:
If going with that solution, why not do it in `componentDidMount` directly?

Reply:
Because I received this strange error, even though the component was theoretically mounted.

![image](https://github.com/opencollective/opencollective-frontend/assets/23138717/7f536ba6-e4ba-4381-898c-9488b9c53da7)

Reply:
Ok, it's likely because of the loading state, the section is not available initially. Then the right approach is:

```suggestion
    if (window.location.hash && !this.componentMounted && !this.props.data.loading) {
      this.componentMounted = true;
```

I'd also recommend renaming `componentMounted` to something more explicit, like `hasTriggeredScroll`.

Then, we still need to link to this section from the relevant places. Have you looked into that?

Reply:
> Then, we still need to link to this section from the relevant places. Have you looked into that?

Do I need to update here, right?

https://github.com/opencollective/opencollective-frontend/blob/97ec80cbc25551112fd652f8de140000b976f81a/components/UserProvider.js#L156